src/js/components/stock-movement-wizard/outbound/PickPage.jsx

```diff
@@ -368,41 +369,41 @@ class PickPage extends Component {
   }
 
   fetchPickPageItems() {
-    const url = `/api/stockMovements/${this.state.values.stockMovementId}/stockMovementItems?stepNumber=4`;
-    apiClient.get(url)
-      .then((response) => {
-        this.setPickPageItems(response, null);
-      });
+    apiClient.get(STOCK_MOVEMENT_ITEMS(this.state.values.stockMovementId), {
+      params: { stepNumber: 4 },
```

Comment:
This `stepNumber` value also could be moved to const mimicking the OutboundWorkflowState enum